cypress/e2e/about.cy.ts

```diff
@@ -2,9 +2,17 @@ import enUS from '../../i18n/locales/en.json'
 
 describe('About page', () => {
     context('Desktop resolution', () => {
+        before(() => {
+            cy.visit("/about")
+            // This wait time is to give the page time to load from Prod when ran in CI.
+            cy.wait(3000)
```

Comment:
adding waits like this as a solution is a really testing bad practice and also compounds as tests grow, however, I don't think this is the right time to build  more elegant loading approaches so this is good for now 👌